src/allmydata/check_results.py

```diff
@@ -1,3 +1,4 @@
+from past.builtins import unicode
```

Comment:
I was thinking about this last night. It seems non-ideal to maintain old names, but I guess the goal is not to remove Python 2 support, but rather to add Python 3 support.

Reply:
This doesn't have to stay this way—this module isn't being ported yet, so trying to do minimal, least risky changes just to get tests elsewhere passing. When it gets ported, yeah, might be worth removing it.

Reply:
> the goal is not to remove Python 2 support, but rather to add Python 3 support.

exactly this